packages/test-helpers/src/osmosis.rs

```diff
@@ -70,24 +156,26 @@ impl<'a> OsmosisVaultRobot<'a, OsmosisTestApp> {
         let treasury = app.init_account(&[]).unwrap();
         let base_pool_id = base_pool.create(app, &admin);
 
-        let code_id = upload_wasm_file(app, &admin, wasm_file_path).unwrap();
+        let contract = ContractType::Artifact(cw_it::Artifact::Local(wasm_file_path.to_string()));
+
+        let code_id = upload_wasm_file(app, &admin, contract).unwrap();
 
         let config = ConfigUnchecked {
             performance_fee: Decimal::percent(3), //TODO: variable performance fee
             treasury: treasury.address(),
-            // TODO: Setup router
-            router: CwDexRouterUnchecked::new(app.init_account(&[]).unwrap().address()),
+            router: dependencies
+                .cw_dex_router_robot
+                .cw_dex_router
+                .clone()
+                .into(),
             reward_assets: vec![AssetInfoUnchecked::native(
                 reward_pool.liquidity[0].denom.clone(),
             )],
             reward_liquidation_target: AssetInfoUnchecked::native(
                 base_pool.liquidity[0].denom.clone(),
             ),
             force_withdraw_whitelist: vec![fwa_admin.address()],
-            // TODO: Setup liquidity helper
-            liquidity_helper: LiquidityHelperUnchecked::new(
-                app.init_account(&[]).unwrap().address(),
-            ),
+            liquidity_helper: dependencies.liquidity_helper.clone().into(),
```

Comment:
dont we need to also set routes ?

Reply:
Good point. I guess our current tests are a bit lacking since they just use reward denoms that are part of the base pool.